include/godot_cpp/core/method_ptrcall.hpp

```diff
@@ -184,6 +184,47 @@ struct PtrToArg<const T *> {
 	}
 };
 
+// Pointers.
```

Comment:
Maybe add a comment that this should be kept in sync with https://github.com/godotengine/godot/blob/master/core/variant/native_ptr.h (especially the `GDVIRTUAL_NATIVE_PTR` calls).